application/src/main/java/com/prjxAPP/Application.java

```diff
@@ -0,0 +1,14 @@
+package com.prjxAPP;
```

Comment:
rename packege from "prjxAPP" to "jungeeks"

Reply:
renamed